### PR TITLE
[chores/docs] Remove HTTP and SSH backends #53

### PR DESCRIPTION
**Pull request by purhan**
Resolves part of #53

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -22,29 +22,36 @@ would become hard to mantain very soon. You also might not want to tie your web
...
+We are currently in 0.1 pre-alpha version.
 
-The 0.1 final version will be out by April 2015.
+The 0.1 final version will be out by August 2021.
+
+.. note::
+
+    The legacy versions of this project had support for SSH and HTTP for extracting information from
+    devices. To see how it worked, visit the
+    `0.1.0 pre-alpha release <https://github.com/openwisp/netengine/releases/tag/0.1.0a>`_ page on
```
Comment (pandafy) [inline docs/source/index.rst:41]: Read the docs provides versioned documentation. If it is not much work to set it up, then we can enable that and use that link here instead.

If above is not quickly doable, then we can link https://github.com/openwisp/netengine/tree/0.1.0a instead of the tag. We can update this link later while releasing the module

File: docs/source/index.rst [hunk 2]
```diff
@@ -22,29 +22,36 @@ would become hard to mantain very soon. You also might not want to tie your web
...
+    github.
 
 =======
 Install
 =======
 
 Install the development version (tarball)::
 
-    pip install https://github.com/ninuxorg/netengine/tarball/master
+    pip install https://github.com/openwisp/netengine/tarball/master
```
Comment (pandafy) [inline docs/source/index.rst:50]: There are few other such occurrence, can you update them too in 

- README
- setup.py
- usage.rst

File: docs/source/index.rst (hunk 1 above)
Comment (pandafy) [reply docs/source/index.rst:41]: Also, it was an alpha release, not pre-alpha.

File: test-settings.example.json [hunk 3]
```diff
@@ -1,20 +1,4 @@
 {
```
Comment (pandafy) [inline test-settings.example.json:1]: Why did you remove this file in this PR?

**Review by pandafy — CHANGES_REQUESTED** [review]
Good work @purhan. Some minor improvements can be done.

File: test-settings.example.json (hunk 3 above)
Comment (purhan) [reply test-settings.example.json:1]: The file is still there, I just removed settings for HTTP and SSH connections.

File: docs/source/index.rst (hunk 1 above)
Comment (pandafy) [reply docs/source/index.rst:41]: Add a note in https://github.com/openwisp/netengine/issues/56, if you want to delegate enabling versioned documentation. So we remember that we need to take care of this.

File: test-settings.example.json (hunk 3 above)
Comment (pandafy) [reply test-settings.example.json:1]: Yes, you are correct. My bad!

File: docs/source/conf.py [hunk 4]
```diff
@@ -44,7 +44,7 @@
 
 # General information about the project.
 project = u'netengine'
-copyright = u'Ninux.org'
+copyright = u'OpenWISP.org'
```
Comment (pandafy) [inline docs/source/conf.py:47]: @nemesisdesign is this okay?

Comment (nemesisdesign) [reply docs/source/conf.py:47]: we are changing this drastically so yes

File: docs/source/index.rst [hunk 5]
```diff
@@ -3,7 +3,7 @@ Netengine
 =========
 
 **Netengine** is a python library that aims to provide a single API to extract common
-information from network devices using different protocols (eg: SNMP, SSH, HTTP) and different firwmares (eg: OpenWRT, AirOS).
+information from network devices using the Simple Network Management Protocol (SNMP) and different firwmares (eg: OpenWRT, AirOS).
```
Comment (nemesisdesign) [inline docs/source/index.rst:6]: it would be nice to mention that implementing backends that retrieve info in other ways is possible if needed.

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
some dependencies like paramiko and mechanize were pulled because of http and ssh backends, make sure the dependencies that are not needed anymore are removed.

File: docs/source/index.rst (hunk 5 above)
Comment (purhan) [reply docs/source/index.rst]: Done, please take a look

**Review by nemesisdesign — APPROVED** [review]
Let's go!
